### ISSUE TITLE
Updater reformatting

### ISSUE THREAD

**Issue by jasonwynn10** [GlitchPlayer/JukeboxPE#15]
Hey, I'm making a website for this repo to host all the versions of the plugin. can I get admin access to the repo so I can make it a github page? I already have the branch created.

**Comment by GlitchPlayer** [comment]
ok

**Comment by GlitchPlayer** [comment]
You already have admin access right?

**Comment by jasonwynn10** [comment]
Nope. I can't see the gear icon for accessing the github pages settings
